test/unit/models/classes/runner/QtiRunnerServiceContinueInteractionTest.php

```
@@ -0,0 +1,581 @@
+        $deliveryExecution->method('getIdentifier')->willReturn($executionId);
+        $deliveryExecution->method('getFinishTime')->willReturn(1234567890.1234);
+        $deliveryExecution->expects($this->once())->method('setState')->willReturn(true);
+
+        $this->injectDeliveryExecutionServiceProxy($deliveryExecution);
+        $this->injectSessionUserUri($userUri);
+
+        $agsPayload = null;
+        $queueDispatcher = $this->createMock(QueueDispatcherInterface::class);
+        $queueDispatcher->expects($this->once())
+            ->method('createTask')
+            ->willReturnCallback(static function ($task, array $body) use (&$agsPayload) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$task'. (undefined)

(UnusedFormalParameter)